spec/support/helpers/file_importer_helper.rb

```diff
@@ -1,9 +1,13 @@
 module Pageflow
-  class FileImporterDummy < Pageflow::FileImporter
+  class FileImporterHelper < Pageflow::FileImporter
```

Comment:
To fit the naming scheme of all other dummy implementations, I'd call this `TestFileImporter`.